src/docs/development/tools/devtools/performance.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Using the Performance view
+description: Learn how to use the DevTools performance view.
+---
+
+## What is it?
+
+The performance view allows you to record and profile a session from your Dart application.
+
+{{site.alert.note}}
+  **If you are running a Flutter application, use a profile build to analyze performance.**
+  Cpu profiles are not indicative of release performance unless your Flutter application is
+  run in profile mode.
+{{site.alert.end}}
+
+## CPU Profiler
+
+Start recording a CPU profile by clicking Record. When you are done recording, click Stop. At this
+point, CPU profiling data is pulled from the VM and displayed in the profiler views (Call Tree,
+Bottom Up, and Flame Chart).
+
+### Profile granularity
+
+The default rate at which the VM collects CPU samples is 1 sample / 250 μs.
+This is selected by default on the Performance view as "Profile granularity: medium".
+This rate can be modified via the selector at the top of the page. The sampling rates
+for low, medium, and high granularity are 1 / 50 μs, 1 / 250 μs, and 1 / 1000 μs,
+respectively. It is important to know the trade-offs of modifying this setting.
+
+A **higher granularity** profile will have a higher sampling rate, and therefore
+a finer-grained CPU profile with more samples. This may also have a performance
+impact on your app since the VM is being interrupted more often to collect samples.
+This will also cause the VM's CPU sample buffer to overflow more quickly. The VM has
+limited space with which it can store CPU sample information. At a higher sampling
+rate, this space will fill up and begin to overflow sooner than it would have if a
+lower sampling rate was used. This means that you may not have access to CPU samples
+from the beginning of the recorded profile.
+
+A **lower granularity** profile will have a lower sampling rate, and therefore will
+yield a coarse-grained CPU profile with fewer samples. However, this comes with a
+lighter performance impact on your app. The VM's sample buffer will also fill less
```

Comment:
sample buffer will also fill less quickly => sample buffer fills more slowly, 
so you will be able to see => so you can see